Common/src/interface_interpolation/CSlidingMesh.cpp

```diff
@@ -286,10 +291,10 @@ void CSlidingMesh::SetTransferCoeff(const CConfig* const* config) {
             }
 
             for(iDim = 0; iDim < nDim; iDim++){
-              donor_iMidEdge_point[iDim] = ( DonorPoint_Coord[ donor_forward_point  * nDim + iDim] +
-                                             DonorPoint_Coord[ donor_iPoint * nDim + iDim] ) / 2;
-              donor_jMidEdge_point[iDim] = ( DonorPoint_Coord[ donor_backward_point * nDim + iDim] +
-                                             DonorPoint_Coord[ donor_iPoint * nDim + iDim] ) / 2;
+              donor_iMidEdge_point[iDim] = ( DonorPoint_Coord[ donor_forward_point ][ iDim] +
+                                             DonorPoint_Coord[ donor_iPoint ][ iDim] ) / 2;
+              donor_jMidEdge_point[iDim] = ( DonorPoint_Coord[ donor_backward_point ][ iDim] +
+                                             DonorPoint_Coord[ donor_iPoint ][ iDim] ) / 2;
```

Comment:
The right thing there would be operator ( , ) because Max replaced pointers with containers.
If we just want to clean the spaces then run clang-format and call it a day.

Reply:
I forgot to replace by `operator(i,j)` here, fixed in 475c6971